src/components/program/TabularView.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Tabs, Tab } from '@edx/paragon';
+import * as ProgramTabs from './tab-components';
+
+
+function TabularView({ programDiscussions }) {
+  const { discussion: { configured, iframe } } = programDiscussions;
+  const courseTabs = [
+    {
+      key: 'JourneyTab', label: 'Journey', isHidden: false, component: <ProgramTabs.JourneyTab />,
+    },
+    {
+      key: 'CommunityTab', label: 'Community', isHidden: !configured, component: <ProgramTabs.CommunityTab iframeComponent={iframe} />,
+    },
+    {
+      key: 'LiveTab', label: 'Live', isHidden: true, component: <ProgramTabs.LiveTab />,
+    },
+  ];
+
+  const [activeTab, setActiveTab] = useState(courseTabs[0].key);
+
+  function changeSelectedTab(tab) {
+    setActiveTab(tab);
+  }
+
+  return (
+    <Tabs activeKey={activeTab.key} id="controlled-tab-example" onSelect={tab => changeSelectedTab(tab)}>
```

Comment:
nit: 
onSelect={tab => changeSelectedTab(tab) => onSelect={tab => setActiveTab(tab).
it approach is better than single liner function.

Reply:
I was doing multiple operations before. on refactoring all those steps are removed, Thanks for pointing out.